config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  resources :dishes, only: [:show, :update]
```

Comment:
Based on comment in controller, consider this option for more RESTful routing for US 2
```
resources :dishes, only: :show do
    resources :ingredients, only: :create, controller: "dish_ingredients"
end
```